article.html

```diff
@@ -0,0 +1,273 @@
+            <!-- Fibonnaci Images -->
+            <div class="image-grid mx-5 my-5 align-items-center">
+              <img class="item-1" src="./images/article-image.jpg" alt="article image">
+              <img class="item-2" src="./images/article-image.jpg" alt="article image">
+              <img class="item-3" src="./images/article-image.jpg" alt="article image">
+              <img class="item-4" src="./images/article-image.jpg" alt="article image">
+            </div>
+          </div>
+        </div>
+        
+        <!-- Article 2nd Half -->
+        <div class="row">
+          <div class="col-lg-12">
+            <p class="lead mx-5 fw-normal text-start">
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec ac luctus orci. Suspendisse ullamcorper purus
+              ligula, id rutrum lorem congue vel. Curabitur malesuada ipsum sagittis velit vehicula dignissim. Donec sagittis
+              ligula at tempor pellentesque. Donec vehicula dui at lacinia ullamcorper. Interdum et malesuada fames ac ante
+              ipsum primis in faucibus. Etiam et commodo libero. Praesent luctus auctor justo, sit amet accumsan nunc feugiat
+              quis. Mauris mattis nunc vel elit posuere, a auctor nunc pulvinar. Integer in eleifend ligula.
+            </p>
+            <p class="lead mx-5 fw-normal text-start">
+              Mauris id eros id eros consectetur lobortis in eget nibh. Sed tortor sem, sollicitudin non est eget, condimentum viverra
+              ligula. Morbi varius libero sed augue mattis lobortis. Fusce mi nunc, convallis vulputate augue sit amet, ullamcorper
+              posuere leo. Quisque a tellus rhoncus, posuere turpis a, elementum magna. Integer elementum tincidunt suscipit. Integer
+              a urna ac sapien convallis vulputate nec quis magna. Nulla facilisi. Phasellus ultricies lectus ac lorem cursus, vel
+              finibus velit sollicitudin. Maecenas congue orci non nisl pellentesque varius. In tempus arcu augue. Vestibulum vel
+              neque vel tortor congue maximus id ut erat. Quisque sed odio diam. Quisque neque erat, rhoncus eu pellentesque id,
+              molestie eget neque.
+            </p>
+          </div>
+        </div>
+        <!-- End of Article -->
+        <div class="row">
+            <div class="container">
+                <div class="row">
+                  <div class="col-12">
+                    <div class="pt-5 pb-5 shadow-sm">
+                      <div class="container">
+                        <div class="row">
+                          <!-- Insert collapse button here-->
+                          <!-- Insert content here -->
+                        </div>
+                      </div>
+                    </div>
+                  </div>
+                </div>
+              </div>
+        </div>
+        </div>
+    </div>
+    <!-- End of News Introduction -->
+    </div>
+</div>
+     <!-- End of News Section -->
```

Comment:
To fix footer-image below:
![image](https://github.com/xkaze09/dpsm-website/assets/47445339/d7834030-80f8-43e6-b41f-ce9622e9400a)
After line 214, add:
```
</div>
</div>
```
Try to see if it works.